main.star

```diff
@@ -1,40 +1,50 @@
-MONGO_DB_PROTOCOL_NAME = "mongodb"
-MONGO_DB_PORT_ID = "mongodb"
+NAME_ARG = "name"
+USER_ARG = "user"
+PASSWORD_ARG = "password"
+IMAGE_ARG = "image"
+ENV_VARS_ARG = "env_vars"
 
-MONGO_DB_DEFAULT_PORT_NUMBER = 27017
-MONGO_DB_PORT_NUMBER_ARG = "MONGO_DB_PORT_NUMBER"
```

Comment:
I intentionally removed this because I would prefer people not need to think about configuring port numbers (one of the value props of Kurtosis)

Reply:
But what happens when there's an application that needs/wants to use a different port? In the Nuvo docker compose they list 27017 as the default, but if the user changes this in their usage then we are stuck?

IMO we shouldn't remove this option